docker/immudb/immudb-config-generator/src/immudb-config-generator.py

```diff
@@ -3,77 +3,42 @@
 from immudb import ImmudbClient
 import immudb.datatypesv2 as datatypesv2
 import urllib.request
-Host=os.environ['IMMUDB_HOST']
```

Comment:
Add  minimal comments on what is happening to make things clear . Also add briefly in readme the purpose and what it is doing